File: app/src/main/java/com/saitejajanjirala/imdbclone/ui/main/MainActivity.java
package com.saitejajanjirala.imdbclone.ui.main;

import androidx.annotation.NonNull;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.content.ContextCompat;
import androidx.fragment.app.Fragment;
import androidx.navigation.fragment.NavHostFragment;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.Menu;
import android.view.MenuInflater;
import android.view.MenuItem;
import android.view.MotionEvent;
import android.view.View;

import com.google.android.material.navigation.NavigationBarView;
import com.saitejajanjirala.imdbclone.R;


import com.saitejajanjirala.imdbclone.databinding.ActivityMainBinding;
import com.saitejajanjirala.imdbclone.ui.home.HomeFragment;

import org.jetbrains.annotations.NotNull;

public class MainActivity extends AppCompatActivity {
    ActivityMainBinding binding;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding= ActivityMainBinding.inflate(LayoutInflater.from(this));
        setContentView(binding.getRoot());
    }


}

File: app/src/main/java/com/saitejajanjirala/imdbclonekotlin/ui/main/MainActivity.kt
package com.saitejajanjirala.imdbclonekotlin.ui.main

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.saitejajanjirala.imdbclonekotlin.R
import dagger.hilt.android.AndroidEntryPoint

@AndroidEntryPoint
class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }
}